src/test/java/Homework18.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+
+
+public class Homework18 extends BaseTest{
+
+    @Test
+ public void PlayaSong(){
+
+        provideEmail("demo@class.com");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        playsong();
+        isSongPlaying();
+
+ }
+public void playsong(){
+    WebElement PlayNextButton = driver.findElement(By.xpath("//*[@title='Play next song']"));
```

Comment:
For variables, the Java naming convention is to always start with a lowercase letter and then capitalize the first letter of every subsequent word.

For example WebElement **playNextButton** = driver.findElement(By.xpath("//*[@title='Play next song']"));